styles/components/_trading-card-image.scss

```diff
@@ -0,0 +1,142 @@
+.cmp-trading-card-image {
+  position: relative;
+  width: fit-content;
+
+  &__overlay {
+    position: relative;
+    height: 350px;
+    z-index: 30;
+
+    &::after {
+      position: absolute;
+      left: 0;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      border-radius: 50%;
+      content: "";
+      transition: opacity 125ms linear;
+    }
+
+    &--gray::after {
+      background-image:
+        linear-gradient(
+          90deg,
+          color("gray-200") -51%,
+          transparent 24%
+        );
+    }
+
+    &--blue::after {
+      background-image:
+        linear-gradient(
+          90deg,
+          color("blue-50-transparency-75") -15%,
+          color("blue-50-transparency-50") 21%,
+          transparent 66%
+        );
+    }
+
+    &--green::after {
+      background-image:
+        linear-gradient(
+          90deg,
+          color("green-50") 20%,
+          color("green-50-transparency-50") 81%,
+          transparent 171%
+        );
+    }
+  }
+
+  &__image {
+    position: relative;
+    height: 350px;
+    border-radius: 50%;
+    overflow: hidden;
+    z-index: 50;
+
+    &::after {
+      position: absolute;
+      left: 0;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      border-radius: 50%;
+      content: "";
+      opacity: 0;
```

Comment:
I think if we remove `opacity: 0`, we should be pretty close to the design for the non-hover state.